end_effectors/ft_sensors/inertials.yaml

```diff
@@ -0,0 +1,12 @@
+ati_mini45:
+  origin:
+    xyz: 0 0 0.01635
+    rpy: 0 0 0
+  mass: 0.163
+  inertia:
+    xx: 0.000070838
+    yy: 0.000070838
+    zz: 0.000107824
+    xy: 0
+    xz: 0
+    yz: 0
```

Comment:
How did you come up with those values?

Reply:
I should've made a comment on this. Mass is measured with a scale, while inertia is computer based on that mass and an STL file

Reply:
Comment added